arch/inst/Zclsd/c.ldsp.yaml

```diff
@@ -0,0 +1,39 @@
+# yaml-language-server: $schema=../../../schemas/inst_schema.json
+
+$schema: "inst_schema.json#"
+kind: instruction
+name: c.ldsp
+long_name: Stack-pointer based load doubleword to even/odd register pair
+description: |
+  Loads stack-pointer relative 64-bit value into registers rd' and rd'+1. It computes its effective
+  address by adding the zero-extended offset, scaled by 8, to the stack pointer, x2. It expands to ld
+  rd, offset(x2). C.LDSP is only valid when rd≠x0; the code points with rd=x0 are reserved.
+definedBy:
+  anyOf: [Zclsd]
+assembly: ld rd, offset(rs1)
+encoding:
+  match: 011-----------01
+  variables:
+    - name: rd
+      location: 11-7
+    - name: imm
+      location: 12|6-2
```

Comment:
The spec has the definitions of the split field as:
- bit 12: offset[5]
- bits 6-2: offset[4:3|8:6]

So, the decode here should be specified as:
```
location: 4-2|12|6-5
left_shift: 3
```